Code/Tools/FBuild/FBuildCore/BFF/Functions/FunctionPrint.cpp

```diff
@@ -124,7 +178,7 @@ FunctionPrint::FunctionPrint()
 
 // PrintVarRecurse
 //------------------------------------------------------------------------------
-/*static*/ void FunctionPrint::PrintVarRecurse( const BFFVariable & var, uint32_t indent )
+/*static*/ void FunctionPrint::PrintVarRecurse( const BFFVariable & var, uint32_t indent, uint32_t maxIndent )
```

Comment:
I went with `maxIndent` instead of a boolean because I thought it would be useful to mimic the behavior of `dt -r2` in windbg, where it will stop expanding members after a certain depth (`maxIndent = indent+2`, for example) or to just constrain how far to the right stuff goes for readability (`maxIndent = 5` or something.)  I didn't end up making those options, but the power is there for the future